api/python/provisioner/__metadata__.py

```diff
@@ -15,8 +15,8 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-__title__ = 'cortx-prvsnr'
-__version__ = '0.36.0'
```

Comment:
That versioning has some meaning. Setting 1.0.0 is not ok currently. Please check https://semver.org/#faq. 
If we decide to set 1.0.0 it should be a common decision and related to some milestone. E.g. GA-1.

Currently it looks like a quick patch. Why do you need that? If you want some specific relation with other packages please may consider to use snapshots in Release: tag. That doc is a quite good set of rules https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_snapshots